src/zope/interface/tests/test_verify.py

```diff
@@ -15,12 +15,29 @@
 """
 import unittest
 
+# pylint:disable=inherit-non-class,no-method-argument,no-self-argument
 
 class Test_verifyClass(unittest.TestCase):
 
-    def _callFUT(self, iface, klass):
+    verifier = None
+
+    @classmethod
+    def setUpClass(cls):
+        # zope.testrunner doesn't call setUpClass, so if you get
+        # 'NoneType is not callable', that's why.
+        cls.verifier = staticmethod(cls._get_FUT())
+
+    @classmethod
+    def _get_FUT(cls):
         from zope.interface.verify import verifyClass
-        return verifyClass(iface, klass)
+        return verifyClass
+
+    _adjust_object_before_verify = lambda self, x: x
```

Comment:
This might be more readable as a regular `def`?

Reply:
I guess that's an idiom I've fallen into. I use it primarily in tests as a signal that it will *definitely* be overridden by subclasses, possibly at runtime. I think it began in abstract base classes as a way to avoid introducing uncovered lines.